Simple-Multiparty/build.gradle

```diff
@@ -15,9 +15,8 @@ buildscript {
 allprojects {
     repositories {
         jcenter()
-        maven {
-            url  "http://tokbox.bintray.com/maven"
-        }
+        maven { url 'https://maven.google.com' }
```

Comment:
I upgraded to the latest appcompat support library: `com.android.support:appcompat-v7:25.4.0`

Google says to add their maven repo to import this: https://developer.android.com/topic/libraries/support-library/setup.html

If I remove this, then I get the compile error: 

`Error:(28, 13) Failed to resolve: com.android.support:appcompat-v7:25.4.0`